docs/admin_guide/user-administration.rst

```diff
@@ -1,22 +1,81 @@
+The *Hops Admin* role, allows all the operations allowed by the *Hops User* role and, in addition to those, it allows users to access the administration panel and manage the Hopsworks platform.
+
+To activate a user account, as an administration, click on the modify icon on the line corresponding to the user you want to activate.
+
+.. _user-admin1.png: ../_images/admin/user-admin1.png
+.. figure:: ../imgs/admin/user-admin1.png
+   :alt: Edit user 
+   :target: `user-admin1.png`_
+   :align: center
+   :figclass: align-cente
+
+   Edit user
+
+This will open a new page with the deatails of the user. In the *Change Settings* section, an administrator can add a role to a user. Expand the drop down menu, select *Hops_User* and click *Add*. If you want to grant admin privileges to the same user, you can repeat the operation this time selecting *Hops_Admin*.
+
+.. _user-admin2.png: ../_images/admin/user-admin2.png
+.. figure:: ../imgs/admin/user-admin2.png
+   :alt: Edit user 
+   :target: `user-admin2.png`_
+   :align: center
+   :figclass: align-cente
+
+   Add role to user
+
+Next to the add user option, there is the *Remove Role* option. By selecting the role and clicking *Remove* an administrator can remove roles from a User.  
+
+Activating users without mail server
+____________________________________
+
+If Hopsworks has been configured to use a mail server/provider, Hopsworks sends out an email to the newly registered user, allowing them to verify their email address. This steps is required before they are allowed to log in.
+
+If however, Hopsworks has not been configured to use a mail server/provider, the verification step can be bypassed by an administrator.
+
+.. _user-admin3.png: ../_images/admin/user-admin3.png
+.. figure:: ../imgs/admin/user-admin3.png
+   :alt: Edit user 
+   :target: `user-admin3.png`_
+   :align: center
+   :figclass: align-cente
+
+   Activate user
+
+To activate an account, an administration can select from the *Change status* dropdown menu the entry *Activated_Account*, and then click *Update* to change the account status.
+
+
+Block/Unblock users
+--------------------
+
+If an administration deems necessary to prevent a user to log in into the platform, they can do so from the same *Change Status* menu shown above. They can select *Blocked_Account* to the prevent the user to log in again.
+
+If a user fails to enter their credential correctly for more than 5 times in a row, their account will be blocked. Users will have to contact an administrator to get their account unblocked. Administrators, from the same UI, will be able to change the account status from *Blocked_Account* back to *Activated_Account*.
+
+Increase users' max number of projects
+--------------------------------------
+
+Users have a limited number of projects they can create. By default it's 5. Please note that project deletion does not decrease the number of projects. This is to prevent abuses of the Yarn quota system.
```

Comment:
Users are only allowed to create a limited number of projects. By default, they can only create 5 projects, but that default number can be changed during installation by ....

Reply:
that deleting a project will not mean that a user can now create more projects. The limit is related to the number of projects a user can create, not the number of projects a user may have created that are currently active. The reason for this approach is to prevent abuses of Yarn's quota system, where a user could continually drop and create projects to acquire new quota.